### ISSUE TITLE
arborescence du site

### ISSUE THREAD

**Issue by emchateau** [ouvroir/patrimoine#2]
Proposition d’arborescence simple, en cherchant à limiter autant que possible le nombre des rubriques. L’utilisation de tags pouvant servir à faciliter la navigation parmi les contenus.

- Guides
- Exemples
- Ressources
- Contribuer
- À propos

## Guides pratiques
- L’engagement numérique sur les RSN
- Exporter et ouvrir ses données
- Fabriquer un numériseur
- Quelles solutions pour publier facilement un site web ?
- Réaliser un audio-guide

## Exemples
- Présentation d’expériences exemplaires
- ITW de personnalités du milieu

## Ressources
- Bibliographie et webographie (avec Zotero ?)
- Soutiens et financement

## Contribuer
- Forum (Liens vers les Issues)
- Comment soumettre une proposition
- Guides du contributeur

**Comment by carlacrz** [comment]
Je suis d'accord avec Alexandra, nous pourrions réaliser des interviews (vidéo ou article) avec les institutions qui ont déjà entrepris des mesures pour diffuser leur collection en ligne si l'une d'elles est d'accord pour partager avec nous sa propre expérience.

Pour ce qui est des guides, j'ajouterai qu'il faudrait insister sur l'importance de la présence sur les réseaux sociaux pour les institutions qui ne l'auraient pas déjà fait ou pour celles qui ne maîtrisent pas parfaitement ces milieux. Nous pourrions faire une sorte de rappel ?

**Comment by Alexandra143** [comment]
Oui bonne idée Carla, peut-être un guide de comment tirer partie des médias sociaux pour l'ouverture de ses contenus et le lien avec ses publics ou quelque chose du genre!

**Comment by emchateau** [comment]
- Page d’accueil : 
comme sur un blog, flux des publications par ordre chronologique
filtres sur la page d’accueil par catégories : sélecteur par domaine, par difficulté, etc.
accès chronologique
- partage des articles avec tags (intégration RSN)

**Comment by emchateau** [comment]
- accueil
- guides
- initiatives
- ressources
- participer
- ?

**Comment by emchateau** [comment]
1. Accueil + actualités
2. Initiatives
3. Tutoriels
4. Ressources
5. Participer
